character cards, social links and tidying

rudytues-day opened the pull request:
- added character cards for aria, css, html and terminal
- added tv tropes links to git & github
- created social icon links
- added contributor card
- moved "PageLayout.astro" into "layouts" directory
- redesigned heart markers
- updated links for consistency

rudytues-day committed:
character cards, social links and tidying

- added character cards for aria, css, html and terminal
- added tv tropes links to git & github
- created social icon links
- added contributor card
- moved "PageLayout.astro" into "layouts" directory
- redesigned heart markers
- updated links for consistency

enigmalea commented on README.md:
```diff
@@ -45,3 +45,13 @@ All commands are run from the root of the project, from a terminal:
 
 Feel free to check [our documentation](https://docs.astro.build) or jump into our [Discord server](https://astro.build/chat).
+
+### rudy tuesday's updates so he can remember
```
In order for us to merge the PR, you'll need to push a commit with these lines removed.

enigmalea commented on src/components/Socials.astro:
```diff
@@ -0,0 +1,36 @@
+---
+
+<ul>
+    <li><a href="https://github.com/fujowebdev/fujowebdev/"><i class="fa-brands fa-github"></i></a></li>
```
These are potentially not screenreader friendly. The logos are not interpreted by most screenreaders and the link names are not being announced. The easiest way around this is to add a title attribute to the href, i.e. `title="Github"`. This results in the screenreader announcing "Github link".

enigmalea commented on src/components/TeamMemberCard.astro:
We should consider if we want to remove the '/' from the end of contact URLs if they're present in order to be consistent. I don't think this is a major deal either way, but it's relatively simple to do and might help clean up our Team page a little more.

essential-randomness replied:
I think cleaning up this page would be good and simple, then we can do a big search and replace later. My preference is for keeping the / wherever possible.

enigmalea commented on src/components/Socials.astro:
```diff
@@ -0,0 +1,36 @@
+    <li><a href="https://blorbo.social/@fujowebdev"><i class="fa-brands fa-mastodon"></i></a></li>
+</ul>
+
+<link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.5.1/css/all.css">
```
Ideally, we should be self-hosting these assets.

@essential-randomness do you have a preference for whether we simply host the css (slightly more complex) or if we import a package like [astro-icon](https://www.astroicon.dev/)

essential-randomness replied:
I think this PR should do whatever is most in line with what we have. Personally, I'd welcome a clean up, and think it would make an excellent bug to file.

enigmalea commented on src/components/Nav.astro:
ms. boba used an image for the github logo in this file, but in our socials section we've used icons provided by fontawesome. Ideally, all of our icons should be from the same source across the project. Once we decide if we are using [astro-icon](https://www.astroicon.dev/) or if we are going to self-host the font-awesome icons ourselves, we should update this logo to match.

essential-randomness replied:
Similarly, good bug to file!

enigmalea reviewed (changes requested):
Excellent start so far! Thank you for your hard work.

enigmalea commented on src/pages/characters.astro:
One other thing that we can leave for another PR, bu if you can figure out how, we should make sure CSS, ARIA, and HTML's names render in all caps to match their official naming within these cards. I don't think this is something that should prevent this merge, but if you can knock it out at the same time that would be great. Otherwise, we can open an issue to tackle this at a later date.

rudytues-day committed:
added titles to social icons, updated readme

- added titles to social icons for accessibility
- made readme less weird
- attempted fix for character name capitalization

rudytues-day reviewed (commented):
made required changes as requested
